zerver/migrations/0177_userprofile_change_emojiset.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+def change_emojiset_choice(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    UserProfile = apps.get_model('zerver', 'UserProfile')
+    for user_profile in UserProfile.objects.exclude(emojiset__in=['google', 'text']).all():
+        user_profile.emojiset = 'google'
+        user_profile.save(update_fields=['emojiset'])
```

Comment:
Let's do this with a bulk query rather than a loop; e.g. `UserProfile.objects.exclude(emojiset__in=['google', 'text']).update(emojiset="google")`.  

The only benefit to doing the query in a loop would be if we needed to clear caches for each user, but (A) there's a better way to do that, and (B) the upgrade/migration process takes care of switching cache prefixes anyway.